src/Symfony/Component/WebLink/WebLinkManagerInterface.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\WebLink;
+
+use Psr\Link\LinkInterface;
+use Psr\Link\LinkProviderInterface;
+
+/**
+ * Manages connections between resources according to the W3C specifications.
+ *
+ * @see https://www.w3.org/TR/html5/links.html
+ * @see https://www.w3.org/TR/resource-hints/
+ * @see https://www.w3.org/TR/preload/
+ * @see http://microformats.org/wiki/existing-rel-values#HTML5_link_type_extensions
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface WebLinkManagerInterface
```

Comment:
would it be possible to have only the PSR interfaces and not add any new in the component? that'd play well with the goal of the psr